.github/workflows/main.yml

```diff
@@ -106,13 +106,17 @@ jobs:
           with:
             requirements: "true"
             test-requirements: "true"
+
         - name: Install spacy language pack
           uses: ./.github/actions/install-spacy-lang
           with:
             pack: en_core_web_sm
-        - name: download wordnet
-          run: python -m nltk.downloader wordnet
 
+        - name: Install nltk wordnet resource
+          run: python -m nltk.downloader wordnet
+          
+        - name: Install nltk punkt resource
```

Comment:
did you find a place where it's required? I was looking through the code and it seems like `punkt` is a vestige of old code, so should not pull it here as well since we don't use it anymore.

Reply:
yes punkt seems to be required by the legacy test suite `test_extra.py`

Reply:
i confirmed that `test_models.py::test_predict` fails if punkt resource is not installed

Reply:
hm, then let's merge it in, but I'll take a closer look later where we use it. Ultimately, it's not a big deal since we need `wordnet` anyways, so installing additional resource is low-lift.